src/providers/BrandingService.js

```diff
@@ -19,14 +19,22 @@ const varMapping = {
   brandImage: 'brand-image-url'
 }
 
-const colorVariants = {
+const brandColorVariations = {
```

Comment:
Please use `colorVariations` as single variation object and use it in both brand and accent